app/views/page-outfit.html

```diff
@@ -3,6 +3,9 @@
   <div id="overview">
     <h1 ng-bind="ship.name"></h1>
     <div id="build">
+      <button ng-click="stripBuild()">
+        <svg class="icon lg"><use xlink:href="#spinner11"></use></svg><span class="button-lbl">Strip</span>
+      </button>
```

Comment:
I'd put the button at the end of the #build div, Such that the flow is:

<Build name> [Save] [Reload] [Delete] [Reset] [Strip]